### ISSUE TITLE
Exception while property mapping for target `array<Neos\Media\Domain\Model\Asset>`

### ISSUE THREAD

**Issue by jonnitto** [neos/neos-development-collection#1593]
### Description

If you add files to a node property with the type `array<Neos\Media\Domain\Model\Asset>`, an exception is thrown

### Steps to Reproduce

1. Install a fresh Neos 
2. Add  a asset list node type
3. Add some files

#### Expected behavior

The files should show up in the inspector.

#### Actual behavior

```
Server communication error: 500 Internal Server Error
Exception while property mapping for target type "array", at property path "0": It is not allowed to map property "__initializer__".It is not allowed to map property "__initializer__".

You need to use $propertyMappingConfiguration->allowProperties('__initializer__') to enable mapping of this property.Exception Code1297759968Exception TypeNeos\Flow\Property\ExceptionLog Reference201705301642130de9deThrown in FileData/Temporary/Development/Cache/Code/Flow_Object_Classes/Neos_Flow_Property_PropertyMapper.phpLine129Original FilePackages/Framework/Neos.Flow/Classes/Property/PropertyMapper.php

Nested ExceptionYou need to use $propertyMappingConfiguration->allowProperties('__initializer__') to enable mapping of this property.Exception Code1335969887Exception TypeNeos\Flow\Property\Exception\InvalidPropertyMappingConfigurationExceptionLog Reference201705301642130312eaThrown in FileData/Temporary/Development/Cache/Code/Flow_Object_Classes/Neos_Flow_Property_PropertyMapper.phpLine186Original FilePackages/Framework/Neos.Flow/Classes/Property/PropertyMapper.php
```

### Affected Versions

Neos: 3.1.1

Flow: 4.1.0

**Comment by kdambekalns** [comment]
Could this be solved with https://github.com/neos/neos-development-collection/pull/1569?

**Comment by jonnitto** [comment]
Hi KarstenI will test it. Later this evening.

**Comment by jonnitto** [comment]
Hi Karsten
Yes, this get's solved with #1569 
Thank you for your fast answer.
